OracleAccessManagement/dockerfiles/buildDockerImage.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Author: Kaushik C
+#
+# Description: script to build a Docker image for Oracle Access Manager 
+#
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+#
+usage() {
+cat << EOF
+
+Usage: buildDockerImage.sh -v [version] -d [domain-type] [-s]
+Builds a Docker Image for Oracle OAM Suite .
+
+Parameters:
+   -h: view usage
+   -v: Release version to build. Required. E.g 12.2.1.4.0
+   -d: Domain type to be built. Required for building the domain oam
+   -s: skips the MD5 check of packages (DEFAULT)
+
+LICENSE CDDL 1.0 + GPL 2.0
+Copyright (c) 2016-2017: Oracle and/or its affiliates. All rights reserved.
+EOF
+exit 0
+}
+
+
+# Validate packages
+checksumPackages() {
+  echo "Checking if required packages are present and valid..."
+  md5sum -c *.download
+  if [ "$?" -ne 0 ]; then
+    echo "MD5 for required packages to build this image did not match!"
+    echo "Make sure to download missing files in folder dockerfiles. See *.download files for more information"
+    exit $?
+  fi
+}
+
+#Parameters
+VERSION="12.2.1.4.0"
+SKIPMD5=1
```

Comment:
Please change the default back to checking MD5 checksums.

Reply:
Updated!!